lib/ui/text.dart

```diff
@@ -1270,6 +1270,9 @@ enum BoxHeightStyle {
     ///
     /// {@macro flutter.dart:ui.boxHeightStyle.includeLineSpacing}
     includeLineSpacingBottom,
+
+    /// Calculate box heights based on the metrics of this paragraph's [StrutStyle].
```

Comment:
This could use a bit more detail about the properties of this, such as each box will have the same height, and the top/bottoms of the boxes will line up with each other.

Reply:
done